lib/python/picongpu/pypicongpu/species/species.py

```diff
@@ -5,14 +5,32 @@
 License: GPLv3+
 """
 
-from ..rendering import RenderedObject
-from .attribute import Attribute, Position, Momentum
-from .constant import Constant, Charge, Mass, DensityRatio, GroundStateIonization, ElementProperties
-from .. import util
+import re
+import typing
+from enum import Enum
 
 import typeguard
-import typing
-import re
+
+from .. import util
+from ..rendering import RenderedObject
+from .attribute import Attribute, Momentum, Position
+from .constant import (
+    Charge,
+    Constant,
+    DensityRatio,
+    ElementProperties,
+    GroundStateIonization,
+    Mass,
+)
+
+
+class Shape(Enum):
+    CIC = "CIC"
+    COUNTER = "Counter"
```

Comment:
```suggestion
    COUNTER = "COUNTER"
```
Maybe an upper case value is easier to deal with?